orderbook/src/main.rs

```diff
@@ -60,6 +60,15 @@ struct Arguments {
     /// Note that if a node does not support the api we still use the less accurate call api.
     #[structopt(long, env = "SKIP_TRACE_API")]
     skip_trace_api: bool,
+
+    /// The amount of time a classification of a token into good or bad is valid for.
+    #[structopt(
+        long,
+        env = "TOKEN_QUALITY_CACHE_EXPIRY",
```

Comment:
for the env var it might be nice to encode the unit (append _sec) so that when someone logs at the staging config they know this is seconds not minutes without having to jump into the codebase.

Reply:
Added SECONDS postfix.